lib/KeyboardAwareHOC.js

```diff
@@ -537,7 +537,7 @@ function KeyboardAwareHOC(
         height: number) => {
 
         let keyboardScreenY = keyboardEndCoordinatesScreenY;
-        let extraScrollHeigth = caretHeight/2; //show half of the bottom line
+        let extraScrollHeigth = caretHeight*2; //show extra space below the caret
```

Comment:
Show extra space when the caret is not at the last line. This gives more "breathing space" between the typing line and the bottom of the screen.

Reply:
Needs checking on landscape mode as well because very little space is left when the keyboard is open

Reply:
we can also consider turning that `factor` into a prop, `extraScrollFactorUsingCaretHeight` ?

Reply:
Needs also checking with big fonts like H2 headings in landscape mode. Not sure if it will fit the screen.